app/styles/app/modules/loading-indicator.scss

```diff
@@ -49,10 +49,6 @@
   padding: 0;
 }
 
-.inline-block {
-  display: inline-block;
-}
-
```

Comment:
Nice, I think it makes a lot of sense to move this out of here and into base.scss 👍